fix(core): remove unused parameter from listener instruction

crisbeto opened the pull request:
Removes the `useCapture` parameter from the `listener` instruction, because it's not used.

pkozlowski-opensource reviewed (approved):
LGTM, thnx for doing this cleanup!
I'm not sure what adev job is complaining about, though...

crisbeto committed:
fix(core): remove unused parameter from listener instruction

Removes the `useCapture` parameter from the `listener` instruction, because it's not used.

pkozlowski-opensource commented:
This PR was merged into the repository by commit 04d963c0a5f68c93015c13973d29107fe35bb942.

The changes were merged into the following branches: main
